standalone-metastore/metastore-common/src/main/java/org/apache/hadoop/hive/metastore/utils/InetUtils.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.hadoop.hive.metastore.utils;
+
+import java.net.InetAddress;
+import java.net.UnknownHostException;
+import java.util.Objects;
+import java.util.Optional;
+
+/**
+ * Utility functions around the Java InetAddress class.
+ */
+public class InetUtils {
+
+  /**
+   * @return name of current host
+   */
+  public static String hostname() {
+    return hostname(Optional.empty());
+  }
+
+  /**
+   * @return name of current host
+   */
+  public static String hostname(Optional<String> defaultValue) {
```

Comment:
I don't see much sanity in working on a machine which doesn't have its hostname set up correctly...I think suppressing issues like this might just hit back later....

I know it's not in the scope of this patch to fix this - but I think we may just forget those `UNKNOWN` -s instead of the hostname. What do you think?

Reply:
@kgyrtkirk Thanks so much for looking at this and thanks for pointing out some silly mistakes on my part.  I must not have had my head on straight that day.

I think we make a functional change as you have proposed in a stand-alone ticket.  Thanks!